### ISSUE TITLE
The dispatcher doesn't log the missing queue name when the destination queue could not be found 

### ISSUE THREAD

**Issue by mauroservienti** [Particular/NServiceBus.AmazonSQS#1678]
The following is an excerpt from a ServiceControl log. An audit instance needs to send messages to a primary instance. The primary instance queue was not available. From the error message, it's impossible to understand what the missing queue is:

```
2022-10-24 14:15:15.9512|21|Error|NServiceBus.Transport.SQS.MessageDispatcher|Exception from Send.
Amazon.SQS.Model.QueueDoesNotExistException: The specified queue does not exist for this wsdl version. ---> Amazon.Runtime.Internal.HttpErrorResponseException: The remote server returned an error: (400) Bad Request. ---> System.Net.WebException: The remote server returned an error: (400) Bad Request.
```

**Comment by danielmarbach** [comment]
Will be fixed as part of 5.6.2

**Comment by danielmarbach** [comment]
[5.6.2 was released and announced](https://discuss.particular.net/t/nservicebus-amazonsqs-5-6-2-patch-release-available/3421)
